ACN to valory + docs (#95)
valory-xyz/open-aea: docs/p2p-connection.md

> The `open_aea/p2p_libp2p:0.26.0` connection allows AEAs to create a peer-to-peer communication network. In particular, the connection creates an overlay network which maps agents' public keys to IP addresses.

Reviewer: ```suggestion
The `valory/p2p_libp2p:0.26.0` connection allows AEAs to create a peer-to-peer communication network. In particular, the connection creates an overlay network which maps agents' public keys to IP addresses.
```

> The `open_aea/p2p_libp2p:0.26.0` connection allows AEAs to create a peer-to-peer communication network. In particular, the connection creates an overlay network which maps agents' public keys to IP addresses.

Reviewer: Care as you changed version on other branch

## Local demo

<div class="admonition note">
  <p class="admonition-title">Note</p>
  <p>Currently <code>p2p_libp2p</code> does not support the 
  ethereum keys generated by open-aea-ledger-ethereum yet.
  This does not affect the agents ability to interact with the 
  ethereum ledger and solely necessitates a separate set of 
  cosmos keys for establishing the libp2p connection.
</p>
</div>

First, make sure you have installed the crypto plugin
of the target test-net. 
``` bash

> pip install open-aea-ledger-cosmos

Reviewer: & open-aea-ledger-ethereum

```

### Create and run the genesis AEA

Create one AEA as follows:

``` bash
aea create my_genesis_aea
cd my_genesis_aea
aea add connection open_aea/p2p_libp2p:0.26.0
aea config set agent.default_connection open_aea/p2p_libp2p:0.26.0
aea install
aea build
```

Establish the <a href="../por">proof of representation</a>:

``` bash

> aea generate-key cosmos

Reviewer: ```suggestion
aea generate-key ethereum
```

> aea add-key cosmos cosmos_private_key.txt

Reviewer: ```suggestion
aea add-key ethereum ethereum_private_key.txt
```
we want ethereum key skill-side

aea generate-key cosmos cosmos_connection_private_key.txt
aea add-key cosmos cosmos_connection_private_key.txt --connection
aea issue-certificates
```

Run the AEA:

``` bash
aea run --connections open_aea/p2p_libp2p:0.26.0
```

Once you see a message of the form `To join its network use multiaddr 'SOME_ADDRESS'` take note of the address. (Alternatively, use `aea get-multiaddress fetchai -c -i fetchai/p2p_libp2p:0.26.0 -u public_uri` to retrieve the address.)
This is the entry peer address for the local <a href="../acn">agent communication network</a> created by the genesis AEA.

### Create and run another AEA

Create a second AEA:

``` bash
aea create my_other_aea
cd my_other_aea
aea add connection open_aea/p2p_libp2p:0.26.0
aea config set agent.default_connection open_aea/p2p_libp2p:0.26.0
aea install
aea build
```

Establish the <a href="../por">proof of representation</a>:

``` bash
aea generate-key cosmos

> aea add-key cosmos cosmos_private_key.txt

Reviewer: see abovde
